nautobot/docs/user-guides/ip-address-merge-tool.md

```diff
@@ -0,0 +1,57 @@
+# Duplicate IP Address Merge Post Migration Tool
+
+Upgrading from Nautobot v1.x to v2.0 can leave many duplicate `IP addresses` in the existing database. This tool is designated to help users to reduce unnecessary duplicate `IP Addresses` by merging them into a single IP Address with all the desired attributes.
+
+## When to use this tool
+
+After upgrading to Nautobot v2.0 and running the data migrations necessary, some unnecessarily duplicate `IP Addresses` might exist in your database. We define duplicate `IP Addresses` as those which have the same `host` attribute but exist in different `namespaces`. If you have no use case to keep those duplicate `IP Addresses` around, we recommend you to use this tool to de-duplicate those `IP Addresses` and keep your database clean and manageable. But if you do have reasons to maintain duplicate `IP Addresses`, this tool is not for you.
+
+!!! important
+    Possible reasons to maintain duplicate `IP Addresses` can be the following:
+    1. You have use cases for duplicate `IP Addresses` with different `nat_inside` attributes.
+    2. Duplicate `IP Addresses` are `primary ips` on different `Devices`.
+
+## How to access this tool
+
+To use this tool:
+1. Go to the `IPAM` tab on the navigation menu and click on `IP Addresses`. This will take you to the list of `IP Addresses` exist in the database.
+2. On the top right of the page, you will find a group of buttons representing different available actions and the first one should be `Find and Merge Duplicate IPs`.
+3. Click on that button to access the tool.
+
+![Homepage Navigation](./images/ip-address-merge-tool/navigation.png)
+
+![Merge Button](./images/ip-address-merge-tool/ip_merge_button.png)
+
+
+## How to use this tool
```

Comment:
Preceding this section I think it would be good to have a simple "How it works" section that explains the basics covered across some of the lower sections. Specifically we should call out:

- The `IPAddress` found in the chosen `Namespace` you select will be the one used as the base object for the others to be merged into
- If you deselect all other duplicates and attempt to merge, nothing will happen (the operation is a no-op) and the object will be skipped over as if you had clicked the "Skip and Go to the Next Duplicates" button.
- All `Interface` assignments will be automatically updated to point to the merged `IPAddress` object
- All `VMInterface` assignments will be automatically updated to point to the merged IPAddress object
- All `Service` assignments will be automatically updated to point to the merged `IPAddress` object
- All duplicate `IPAddress` objects will be deleted after being merged, leaving only the merged `IPAddress` object in the `Namespace` you selected

And any others?